dateparser/timezones.py

```diff
@@ -6,7 +6,7 @@
 timezone_info_list = [
     {
         'regex_patterns':
-            [r'(\W|\d|_)\(?%s\)?$'],
+            [r'(\W|\d|_)\(?%s(\)|$|[^\w+])'],
```

Comment:
Like visible from the `date_parser.py` code in the comment above, we don't translate date at this level. Firstly parenthesis wrapped timezone issue was reported [here](https://github.com/scrapinghub/dateparser/issues/45)